jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/MemoryMXBeanImpl.java

```diff
@@ -777,33 +777,49 @@ public String getGCMode() {
 	private native String getGCModeImpl();
 
 	/**
-	 * Returns the amount of CPU time spent in the GC by the master thread, in milliseconds.
+	 * Returns the amount of CPU time spent in the GC by the main thread, in milliseconds.
 	 *
 	 * @return CPU time used in milliseconds
-	 * @see #getGCMasterThreadCpuUsed()
+	 * @see #getGCMainThreadCpuUsed()
 	 */
-	private native long getGCMasterThreadCpuUsedImpl();
+	private native long getGCMainThreadCpuUsedImpl();
 
 	/**
 	 * {@inheritDoc}
 	 */
+	@Deprecated
```

Comment:
Pls use the long form for Java 11+, i.e. `@Deprecated(forRemoval=true, since="15")`.
You can create the processor instructions for Java 16 and remove the methods for Java 16+.

Reply:
I have updated the deprecated annotation and added preprocessor instructions to remove the methods in Java16.